woocommerce/woocommerce
# Simplify the product images empty state

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #45920



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the new product editor is enabled under the WooCommerce > Settings > Advance > Features
2. Go to Products > Add new
3. From the General tab the Images section should looks like the image below when it's empty 
![image](https://github.com/woocommerce/woocommerce/assets/13334210/4fd5f0a7-9bf4-4eeb-9020-1cfcc69e869e)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
